middleware/v1/body.go

```diff
@@ -0,0 +1,58 @@
+package middleware
+
+import (
+	"context"
+	"encoding/json"
+	"net/http"
+
+	"github.com/opensaucerer/gotemplate/helper"
+	"github.com/opensaucerer/gotemplate/typing"
+)
+
+//BodyH is a middleware that takes a struct and a request handler. It parses
```

Comment:
Please let the comment be structured the way I structured it in my comment.

Reply:
Done.
I apologize, I didn’t see the structure you sent early, I only got the comment in original text.